test/adapter/json_api/links_test.rb

```diff
@@ -13,6 +13,10 @@ class LinkAuthorSerializer < ActiveModel::Serializer
             end
 
             link :other, '//example.com/resource'
```

Comment:
Agreed, however this is not a relative link. `//resource` would be undefined, but `//example.com/resource` is a valid url.